docs/plugins.yml

```diff
@@ -68,6 +68,7 @@
 - intelsdi-x/snap-plugin-collector-snmp
 - intelsdi-x/snap-plugin-collector-smart
 - intelsdi-x/snap-plugin-collector-swap
+- intelsdi-x/snap-plugin-collector-use
```

Comment:
Is the plugin name an acronym for something? it doesn't seem really clear what it's collecting.